packages/docs/docs/02-Guides/02-Reading/04-Refresh Per Blocks.mdx

```diff
@@ -0,0 +1,67 @@
+# Refresh Per Blocks
+
+### Never
+
+To disable refreshing calls per block, just pass `refresh` parameter as `never` to config, it will fetch data only once.
+
+**Example**
+
+```tsx
+  import { Mainnet } from '@usedapp/core'
+
+  const config = {
+    readOnlyChainId: Mainnet.chainID,
+    readOnlyUrls: {
+      [Mainnet.chainID]: 'https://mainnet.infura.io/v3/62687d1a985d4508b2b7a24827551934',
+    },
+    refresh: 'never',
```

Comment:
I suppose the common practice in such cases is to set parameter to a negative value, such as -1.